ios-base/build_kotlin_modules.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd ../
```

Comment:
`cd "$(git rev-parse --show-toplevel)"` sounds better. It allows moving up to the repository root directory.

Reply:
This is no longer needed by https://github.com/DroidKaigi/conference-app-2020/pull/447/commits/2111bd7f13b0ac2963a14c516488dd49aeacc75d 🙏